app-utils/bili-sync/spec

```diff
@@ -0,0 +1,4 @@
+VER=2.1.1
+SRCS="git::commit=tags/v$VER::https://github.com/amtoaer/bili-sync"
+CHKSUMS="SKIP"
+CHKUPDATE="github::repo=amtoaer/bili-sync"
```

Comment:
Please use Anitya where possible, as the `github::` type would require an API key.